cmd/operator-sdk/olmcatalog/gen-csv.go

```diff
@@ -61,7 +61,7 @@ Configure CSV generation by writing a config file 'deploy/olm-catalog/csv-config
 		log.Fatalf("Failed to mark `csv-version` flag for `olm-catalog gen-csv` subcommand as required: %v", err)
 	}
 	genCSVCmd.Flags().StringVar(&fromVersion, "from-version", "", "Semantic version of an existing CSV to use as a base")
-	genCSVCmd.Flags().StringVar(&csvConfigPath, "csv-config", "", "Path to CSV config file. Defaults to deploy/olm-catalog/csv-config.yaml")
+	genCSVCmd.Flags().StringSliceVar(&excludePaths, "exclude", nil, "Paths to exclude from CSV generation, ex. \"deploy/prod,deploy/test\"")
```

Comment:
I think should be e.g instead of e.x . Am I right?

Reply:
I think would be good shows as example it passing 2 values since the users by do not know how to split them. I mean what character should be used to inform more than one.

Reply:
e.g. vs ex. is a matter of preference.

The example has two values: `deploy/prod` and `deploy/test`. The format of the example is exactly how you would pass a string array to the CLI.